apps/files/src/components/AllFilesList.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="uk-height-1-1">
+    <div class="uk-flex uk-flex-column uk-height-1-1">
+      <div id="files-list-container" class="uk-overflow-auto uk-flex-auto">
+        <file-list :fileData="fileData" id="files-list" :loadingFolder="loadingFolder" :actions="actions" :compactMode="_sidebarOpen"
+            :isActionEnabled="isActionEnabled">
+          <template #headerColumns>
+            <oc-table-cell shrink type="head" v-if="!publicPage()" />
+            <oc-table-cell type="head" class="uk-text-truncate" v-translate>Name</oc-table-cell>
+            <oc-table-cell shrink type="head" :class="{ 'uk-visible@s' : !_sidebarOpen, 'uk-visible@m'  : _sidebarOpen }"><translate>Size</translate></oc-table-cell>
+            <oc-table-cell shrink type="head" :class="{ 'uk-visible@s' : !_sidebarOpen, 'uk-visible@m'  : _sidebarOpen }" class="uk-text-nowrap" v-translate>Modification Time</oc-table-cell>
+          </template>
+          <template v-slot:rowColumns="{ item }">
```

Comment:
Get rid pls of `v-slot`. https://vuejs.org/v2/style-guide/#Directive-shorthands-strongly-recommended